app/routes/apply/eligibility/claim/about-child.js

```diff
@@ -4,20 +4,40 @@ const referenceIdHelper = require('../../../helpers/reference-id-helper')
 const ValidationError = require('../../../../services/errors/validation-error')
 const insertChild = require('../../../../services/data/insert-child')
 
+const REFERENCE_SESSION_ERROR = '?error=expired'
+
 module.exports = function (router) {
-  router.get('/apply/:claimType/eligibility/:referenceId/claim/:claimId/child', function (req, res) {
+  router.get('/apply/eligibility/claim/about-child', function (req, res) {
     UrlPathValidator(req.params)
 
+    if (!req.session ||
+        !req.session.claimType ||
+        !req.session.referenceId ||
+        !req.session.decryptedRef ||
+        !req.session.advanceOrPast ||
+        !req.session.claimId) {
```

Comment:
this check is repeated in a lot of places, we should factor it out into a SessionHandler module which checks that the session object has the right params at each stage. This could also be used to clear the session at start and end.